pyanaconda/modules/network/installation.py

```diff
@@ -179,16 +180,23 @@ def _write_interface_rename_config(self, root, ifname_option_values, overwrite):
         :param overwrite: overwrite existing configuration file
         :type overwrite: bool
         """
+
+        if ifname_option_values:
+            target_system_dir = os.path.normpath(root + self.SYSTEMD_NETWORK_CONFIG_DIR)
```

Comment:
We try to use `join_paths` now, but it is up to you.

Reply:
Used.